src/test/compile-fail/borrowck/borrowck-local-borrow-outlives-fn.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// revisions: ast mir
+//[mir]compile-flags: -Z emit-end-regions -Z borrowck-mir
```

Comment:
You don't need the emit-end-regions flag any more btw, thanks to this very nice PR: #44717. My PR #46106 is removing them everywhere but I guess I should start spreading the word so that the line doesn't keep getting added :p.

The comment applies to the other places where you have -Z emit-end-regions as well, but I won't comment on each of them.